HW1/Task_2.py

```diff
@@ -0,0 +1,23 @@
+#3. Напишите код, который запрашивает число и сообщает является ли оно простым или составным.
+#Используйте правило для проверки: “Число является простым, если делится нацело только на единицу и на себя”.
+# Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+
+MAX_LIMIT = 100000
+
+while True:
+    print(f'Веедите число между 1 и {MAX_LIMIT}')
+    num = int(input())
+    if num < 0 or num > MAX_LIMIT:
+        print(f'Число {num} не в границах интервала 0 и {MAX_LIMIT}')
+    else:
+        break
+
+count = 0
+for i in range(1, num+1):
+    if num % i == 0:
+        count += 1
+
+if count <= 2:
+    print(f'число {num} простое')
+else:
+    print(f'число {num} составное')
```

Comment:
Можно обойтись без переменной `count`
```suggestion
for i in range(1, num+1):
    if num % i == 0:
        print(f'число {num} составное')

else:
    print(f'число {num} простое')
```